### PR TITLE
Filip/soc2 gn 80

### PR DESCRIPTION
**Pull request by filiphaftek**
Documentation for:

1. [Disaster recovery process](https://github.com/sourcegraph/sourcegraph/issues/29601)
2. [Annual backup and restore of the databases and configuration files](https://github.com/sourcegraph/sourcegraph/issues/29592)

### CODE REVIEW

**Comment by daxmc99** [comment]
We should also update this playbook https://handbook.sourcegraph.com/departments/product-engineering/engineering/process/incidents/playbooks/dotcom_deleted_entirely

**Comment by danieldides** [comment]
Create a link to this disaster recovery page from the `Playbooks` page: https://handbook.sourcegraph.com/departments/product-engineering/engineering/process/incidents/playbooks

Probably should just make a `Cloud DevOps Playbooks` section that we can just link into from the `incidents/playbooks` landing page.

File: content/departments/product-engineering/engineering/cloud/devops/disaster-recovery.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+    - [On demand backup steps](#on-demand-backup-steps)
+    - [Scheduled backup steps](#scheduled-backup-steps)
+  - [GKE Kubernetes restore process](#gke-kubernetes-backup-process)
+  - [Update nginx ingress controller](#update-nginx-ingress-controller)
+
+# Cloud base infrastructure
+
+[Sourcegraph](https://sourcegraph.com) is deployed in Google Cloud Platform.
+
+![Sourcegrap.com in Gogole Cloud Platform](sourcegraph_cloud.png)
```
Comment (daxmc99) [inline content/departments/product-engineering/engineering/cloud/devops/disaster-recovery.md]:
```suggestion
![Sourcegrap.com in Google Cloud Platform](sourcegraph_cloud.png)
```

**Comment by danieldides** [comment]
Small typo in the chart. `sesnsitive` -> `sensitive`.

**Comment by filiphaftek** [comment]
> We should also update this playbook https://handbook.sourcegraph.com/departments/product-engineering/engineering/process/incidents/playbooks/dotcom_deleted_entirely

I think that to put these procedures into playbooks with would have to ensure that we have daily DB exports to GC bucket and Velero daily backup.

**Comment by filiphaftek** [comment]
> Create a link to this disaster recovery page from the `Playbooks` page: https://handbook.sourcegraph.com/departments/product-engineering/engineering/process/incidents/playbooks
> 
> Probably should just make a `Cloud DevOps Playbooks` section that we can just link into from the `incidents/playbooks` landing page.

This page is strictly for annual DR process and SOC2, but we could take parts of it and make short FAQ/playbooks in the incidents landing page 👍

**Review by daxmc99 — APPROVED** [review]
LGTM
Planning to merge this

**Pull request merged**